app/controllers/api/v1/wallets_controller.rb

```diff
@@ -0,0 +1,77 @@
+class Api::V1::WalletsController < ApplicationController
+  before_action :authenticate_request!
+  before_action :find_wallet, only: [:show, :update, :destroy]
+
+  def index
+    @wallets = []
+    current_user.user_wallets.each do |user_wallet|
+      @wallets.push(user_wallet.wallet)
+    end
+    render :index, status: :ok
+  end
+
+  def create
+    @wallet = Wallet.new(create_params)
+
+    if @wallet.save
+      UserWallet.create(
+        user_id: current_user.id,
+        wallet_id: @wallet.id,
+        user_role: User::roles[:OWNER]
```

Comment:
```suggestion
        user_role: User.roles[:OWNER]
```
https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/ColonMethodCall